pyquil/quilbase.py

```diff
@@ -145,6 +145,8 @@ def _convert_to_rs_instruction(instr: Union[AbstractInstruction, quil_rs.Instruc
         return quil_rs.Instruction.from_gate_definition(instr)
     if isinstance(instr, Halt):
         return quil_rs.Instruction.new_halt()
+    if isinstance(instr, Include):
```

Comment:
Is there any way to introduce a CI error if one of the valid instruction types is not covered?

Reply:
Not that I could figure out. If `quil_rs.Instruction` was a true Python enum, we might be able to use [assert_never](https://typing.readthedocs.io/en/latest/source/unreachable.html#assert-never-and-exhaustiveness-checking), but since it's more of an enum-like wrapper, I don't think the type system can figure out how to do an exhaustiveness check against it.